wordpress_importer/tests/test_import_command.py

```diff
@@ -57,7 +58,6 @@ def local_get_404(url):
     settings.PROJECT_ROOT)
 
 
-@mock.patch('requests.get', local_get_successful)
```

Comment:
Had to move this to the individual functions, as one of them needs a different patch.  Just putting the override on the single function didn't work.